.github/workflows/prebuild.yaml

```diff
@@ -50,10 +64,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [21]
+        node: [latest]
         canvas_tag: ["v3.0.0-rc2"] # e.g. "v2.6.1"
     name: ${{ matrix.canvas_tag }}, Node.js ${{ matrix.node }}, Linux
     runs-on: ubuntu-latest
+    if: contains(fromJson(inputs.os), 'ubuntu-latest')
```

Comment:
GitHub Actions docs on detecting the operating system: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#detecting-the-operating-system
```
- if: runner.os == 'Linux'
- if: runner.os == 'macOS'
- if: runner.os == 'Windows'
- if: runner.os != 'Windows'
```